projects/driver_controls/inc/stm32f0xx/ads1015_def.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#define ADS1015_I2C_ADDRESS_GND ((uint8_t)0x48)
+
+#define ADS1015_ADDRESS_POINTER_CONV ((uint8_t)0x0)
+#define ADS1015_ADDRESS_POINTER_CONFIG ((uint8_t)0x1)
+#define ADS1015_ADDRESS_POINTER_LO_THRESH ((uint8_t)0x2)
+#define ADS1015_ADDRESS_POINTER_HI_THRESH ((uint8_t)0x3)
+
+#define ADS1015_LO_THRESH_REGISTER_MSB ((uint8_t)0x0)
+#define ADS1015_LO_THRESH_REGISTER_LSB ((uint8_t)0x0)
+#define ADS1015_HI_THRESH_REGISTER_MSB ((uint8_t)0xFF)
+#define ADS1015_HI_THRESH_REGISTER_LSB ((uint8_t)0xFF)
+
+#define ADS1015_NO_EFFECT ((uint8_t)0x0 << 7)
```

Comment:
What does "no effect" mean?

Reply:
I will take that out since it does not do anything if you write that byte to the register.